client/src/About.css

```diff
@@ -0,0 +1,31 @@
+.about-card-wrapper {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    padding: 40px 0 60px 0;
+    gap: 10px;
+}
+  
+@media only screen and (min-width: 375px) {
+    .about-card-wrapper {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      text-align: center;
+      padding: 100px 0 20px 0;
+      gap: 10px;
+      margin: 220px auto 60px auto;
+    }
+}
+
+.about-link {
+    padding-top: 10px;
+    color: #e94de9;
```

Comment:
If you find you re-use colours, CSS variables can be a good way to give them readable names so you don't have to remember which hex value is which colour

Reply:
Yes, it's true. I have used them in other projects. I will consider adding them to this one as well. It totally makes sense right. 👍